portal-web/test/functional/com/liferay/portalweb/tests/enduser/changelist/ChangeListManagePublications.testcase

```diff
@@ -0,0 +1,95 @@
+@component-name = "portal-change-list"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Change List";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		GlobalMenu.gotoPortlet(
+			category = "Change Lists",
+			panel = "Applications",
+			portlet = "Settings");
+
+		ChangeList.enableChangeList();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			GlobalMenu.gotoPortlet(
+				category = "Change Lists",
+				panel = "Applications",
+				portlet = "Settings");
+
+			ChangeList.disableChangeList();
+		}
+	}
+
+	@priority = "3"
+	test CancelPublicationCreation {
+		property test.name.skip.portal.instance = "ChangeListManagePublications#CancelPublicationCreation";
+
+		ChangeList.cancelPublicationCreation();
+	}
+
+	@description = "Try to create new change list with too long name"
+	@priority = "3"
+	test CreateNewPublicationWithLongName {
+		property test.name.skip.portal.instance = "ChangeListManagePublications#CreateNewPublicationWithLongName";
+
+		ChangeList.createNewPublication(
+			gotoManagePublications = "true",
+			publicationDescription = "Publication Description",
+			publicationName = "Lopadotemachoselachogaleokranioleipsanodrimhypotrimmatosilphioparaomelitokatakechymenokichlepikossyphophattoperisteralektryonoptekephalliokigklopeleiolagoiosiraiobaphetraganopterygon");
+
+		Alert.viewLengthValidationField();
+	}
+
+	@description = "Try to create new change list with empty string in the description"
+	@priority = "3"
+	test CreateNewPublicationWithoutDescription {
+		property test.name.skip.portal.instance = "ChangeListManagePublications#CreateNewPublicationWithoutDescription";
+
+		ChangeList.createNewPublication(
+			gotoManagePublications = "true",
+			publicationName = "Publication Name");
+
+		AssertTextPresent(
+			locator1 = "ChangeList#CHANGE_LIST_INDICATOR_TITLE",
+			value1 = "Publication Name");
+	}
```

Comment:
To simplify things, can we just have one test add a description and the others don't?